capstone_project_EEG_detection.ipynb

```diff
@@ -16,80 +16,46 @@
   },
   {
```

Comment:
this should really all be in python files, there is no reason all this data processing should be in a notebook; this makes the review process more convoluted than it should be, risking missing important checks and balances

Reply:
Most likely, yeah. Would be nice to have maybe a data_formatting.py file containing all these functions and a data_processing.py containing the supervisor-provided code that does the wavelet transforms. It's also important that we note that those functions aren't our work, so having it in a separate file aids with that.